WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -205,6 +205,62 @@
         <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| "#{c[:name]} (#{c[:iso_code]})" }, value_method: lambda { |c| c[:iso_code] }, disabled: disable_money_input  %>
         <%= f.input :base_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, disabled: disable_money_input, currency_selector: "#competition_currency_code" %>
 
+        <script>
+          var bandFee = 0;
+          var baseFee = 0;
+          var compLimit = null;
+          var tempLimit = null;
+
+          <% countryBand = CountryBand.where(iso2: @competition.serializable_hash[:country_iso2]).map(&:number)[0] %>
+          <% if !countryBand.nil? && countryBand > 0 %>
+            bandFee = <%= CountryBand::BANDS[countryBand][:value] %>;
+          <% end %>
+
+          function UpdateDues(){
+            var pricePerCompetitor = Math.max(bandFee, baseFee) * 100;
+            window.wca.cancelPendingAjaxAndAjax('dues_formatting', {
+              url: '<%= dues_formatting_path %>',
+              data: {
+                value: compLimit == null ? pricePerCompetitor : pricePerCompetitor * compLimit,
+                to_currency: "<%= @competition.currency_code %>",
+              },
+              success: function(data) {
+                var dues = data.html;
+                if(compLimit != null)
+                  $('#dues_estimate').html("<b>" + I18n.t('competitions.competition_form.dues_estimate.calculated', {limit: compLimit, estimate: dues}) + "</b>");
+                else
+                  $('#dues_estimate').html("<b>" + I18n.t('competitions.competition_form.dues_estimate.per_competitor', {estimate: dues}) + "</b>");
+              }
+            });
+          }
+
+          $('input[name="competition[base_entry_fee_lowest_denomination]"]').on('change', function() {
+            if(!$('#dues_estimate').length){
+              $(this).siblings('.help-block').append('<br><p id="dues_estimate"><b></b></p>');
+            }
+            baseFee = this.value / 100 * 0.05; //TODO: check our currency subunit_to_unit
+            UpdateDues();
+          }).trigger('change');
+
+          $('select[name="competition[competitor_limit_enabled]"]').on('change', function() {
+            if(this.value === "false"){
```

Comment:
Is the state of a checkbox really encoded as string in JS?!?

Reply:
yes... I see `=== "true"` and `=== "false"` used in other jquery files so I kept things consistent

Reply:
Okay. This fact seems really weird to me (not your fault at all) but I appreciate that you're trying to keep things consistent. So that's fine by me :D